### DISCUSSION TITLE
开工开工 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#139] [Announcements]
https://blog.goalonez.site/blog/%E5%BC%80%E5%B7%A5%E5%BC%80%E5%B7%A5.html 

 Goalonez Blog

**Comment by luckyscript** [comment]
博客有RSS么

**Comment by Goalonez** [comment]
> @luckyscript
> 博客有RSS么

暂时还没有

**Comment by Goalonez** [comment]
> @luckyscript
> 博客有RSS么

已添加RSS：https://blog.goalonez.site/feed.xml
